pwndbg/auxv.py

```diff
@@ -138,15 +141,15 @@ def find_stack_boundary(addr):
     addr = pwndbg.lib.memory.page_align(int(addr))
     try:
         while True:
-            if b"\x7fELF" == pwndbg.gdblib.memory.read(addr, 4):
+            if b"\x7fELF" == bytes(pwndbg.gdblib.memory.read(addr, 4)):
```

Comment:
It's fine to compare bytes with bytearray (returned by .read) so this cast is not needed
```suggestion
            if b"\x7fELF" == pwndbg.gdblib.memory.read(addr, 4):
```

Reply:
fixed!